### PR TITLE
Update product-technical-writing-team-handbook.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Fixed emphasis formatting, rectified incorrect links, and added some additional content around Contributing and doc MVPs.

### CODE REVIEW

File: operations/research-and-development/product/product-technical-writing-team-handbook.md [hunk 1]
```diff
@@ -40,29 +40,48 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
...
+### DocUp Issues
 
 Requests for documentation can be made within [community.mattermost.com](https://community.mattermost.com), using the DocUp plugin embedded in the post menu.
 
 ![Access the Doc Up plugin by hovering over a message and clicking the &quot;...&quot; menu.](../../../.gitbook/assets/image%20%2818%29.png)
 
-When you select DocUp, an issue is generated in the GitHub docs repo, and added to the issues list.
+When you select DocUp, an issue is generated in the GitHub docs repo, and added to the issues list. An update is listed in the Documentation channel, with the issue link.
+
+As the issues are open to the community, the more information provided in the ticket the better. Once the link is shared in the Documentation channel, review the issue and add any links to appropriate documentation and/or Jira tickets. This ensures that the assignee is able to take on and complete the work within the turnaround times.
```
Comment (jasonblais) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md]: Should they review it in the repository instead?

Also, depending on the option you choose, the issue may be created in mattermost-handbook or mattermost-developer-documentation repos

File: operations/research-and-development/product/product-technical-writing-team-handbook.md [hunk 2]
```diff
@@ -40,29 +40,48 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
...
-### DocUp/Documentation Issue Process
+### Documentation MVP
+
+We want to empower everyone to contribute to our documentation, and be comfortable submitting documentation for contributions. As such, we don't expect every contribution to adhere to our style guide. 
+
+Here are some guidelines around this:
+
+* When submitting a PR, please include updated documentation if applicable. 
+* The documentation update can be in the form of a bullet list or an outline.
+* Label the PR as **Docs Needed** and tag @justinegeffen or @amyblaise.
```
Comment (jasonblais) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md]:
```suggestion
* Label the PR as **Docs Needed** and tag @justinegeffen or @amyblais.
```

File: operations/research-and-development/product/product-technical-writing-team-handbook.md [hunk 3]
```diff
@@ -40,29 +40,48 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
...
 
 The basic outline for getting started with contributions is provided in the [README](https://github.com/mattermost/docs/blob/master/README.md) of the docs repo. If you have write access to the repo, you can create a branch off master and work on that.
 
-Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review/) - it applies to various types of contributions including documentation.
+Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review) - it applies to various types of contributions including documentation.
 
-### DocUp/Documentation Issue Process
+### Documentation MVP
+
+We want to empower everyone to contribute to our documentation, and be comfortable submitting documentation for contributions. As such, we don't expect every contribution to adhere to our style guide.
```
Comment (jasonblais) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md]: `As such, we don't expect every contribution to adhere to our style guide. ` --> should we reword it differently to emphasize that you or someone else will update the contribution to adhere to our style guide. I don't want to set the wrong expectations about lowering our standards

**Review by jasonblais — CHANGES_REQUESTED** [review]

File: operations/research-and-development/product/product-technical-writing-team-handbook.md (hunk 3 above)
Comment (justinegeffen) [reply operations/research-and-development/product/product-technical-writing-team-handbook.md]: Yep, that's not properly conveyed at all.

How about:
"As such, we don't expect every contribution to adhere to our style guide when first submitted. During the review process the editorial team also provide feedback on style elements to bring the submission in line with the Documentation Style [Guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide)."

File: operations/research-and-development/product/product-technical-writing-team-handbook.md [hunk 4]
```diff
@@ -40,29 +40,48 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
...
+Note: This process does not apply to the [API Documentation requirements](https://github.com/mattermost/mattermost-api-reference). 
+
+### DocUp Issues
 
 Requests for documentation can be made within [community.mattermost.com](https://community.mattermost.com), using the DocUp plugin embedded in the post menu.
 
 ![Access the Doc Up plugin by hovering over a message and clicking the &quot;...&quot; menu.](../../../.gitbook/assets/image%20%2818%29.png)
 
-When you select DocUp, an issue is generated in the GitHub docs repo, and added to the issues list.
+When you select DocUp, an issue is generated in the GitHub docs repo, and added to the issues list. An update is listed in the Documentation channel, with the issue link.
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md]:
```suggestion
When you select Doc Up and choose **Admin** as the issue type, an issue is generated in the GitHub docs repo, and added to the issues list. An update is listed in the Documentation channel, with the issue link. 

You can also select **Developer** or **Company Handbook** to direct the Doc Up  request to the appropriate repo. 
```

File: operations/research-and-development/product/product-technical-writing-team-handbook.md [hunk 5]
```diff
@@ -40,29 +40,48 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
...
+
+As the issues are open to the community, the more information provided in the ticket the better. Once the link is shared in the Documentation channel, review the issue and add any links to appropriate documentation and/or Jira tickets. This ensures that the assignee is able to take on and complete the work within the turnaround times.
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md]:
```suggestion
As the issues are open to the community, the more information provided in the issue the better. 

Review the issue in the [docs repo](https://github.com/mattermost/docs/issues?q=is%3Aopen+is%3Aissue), [developer documentation repo](https://github.com/mattermost/mattermost-developer-documentation/issues?q=is%3Aopen+is%3Aissue), or [handbook repo](https://github.com/mattermost/mattermost-handbook/issues?q=is%3Aopen+is%3Aissue) and add any links to appropriate documentation and/or Jira tickets. This ensures that the assignee is able to take on and complete the work within the turnaround times. 
```

File: operations/research-and-development/product/product-technical-writing-team-handbook.md (hunk 1 above)
Comment (justinegeffen) [reply operations/research-and-development/product/product-technical-writing-team-handbook.md]: Good points; updated.

File: operations/research-and-development/product/product-technical-writing-team-handbook.md [hunk 6]
```diff
@@ -40,29 +40,75 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
...
+* Label the PR as **Docs Needed** and tag @justinegeffen or @amyblais.
+* The documentation you've submitted will be reviewed.
+* At times, the editors may make and commit stylistic changes \(such as punctuation\) but any content changes will be added as a suggestion for the submitter's consideration. 
+* Once the PR is approved, it will be merged, and the documentation will be updated. 
+
+You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review). 
+
+Note: This process does not apply to the [API Documentation requirements](https://github.com/mattermost/mattermost-api-reference). 
+
+### Adding Reviewers to PRs
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md:74]: @jasonblais, I added dev reviewers based on the list here: https://developers.mattermost.com/internal/rd-teams/. Open to suggestions regarding the areas I've linked them to and/or ones that aren't listed but should be.

File: operations/research-and-development/product/product-technical-writing-team-handbook.md [hunk 7]
```diff
@@ -40,29 +40,75 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
...
 
-Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review/) - it applies to various types of contributions including documentation.
+Once complete, submit your Pull Request \(PR\). Ensure that you assign appropriate approvers and labels. You can read more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review) - it applies to various types of contributions including documentation.
 
-### DocUp/Documentation Issue Process
+### Submitting Documentation With Your PR
 
-Requests for documentation can be made within [community.mattermost.com](https://community.mattermost.com), using the DocUp plugin embedded in the post menu.
+We want to empower everyone to contribute to our documentation, and be comfortable submitting documentation for contributions. As such, we don't expect every contribution to adhere to our style guide.
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md]:
```suggestion
We want to empower everyone to contribute to our documentation, and be comfortable submitting documentation for contributions. As such, we don't expect every contribution to adhere to our style guide when first submitted. 

During the review process the editorial team also provides feedback on style elements to bring the submission in line with the [Documentation Style Guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide).
```

File: operations/research-and-development/product/product-technical-writing-team-handbook.md (hunk 6 above)
Comment (jasonblais) [reply operations/research-and-development/product/product-technical-writing-team-handbook.md:74]: That's a good start, we can have Dean Whillier as the dev reviewer for `End User Guide - Other: `

File: operations/research-and-development/product/product-technical-writing-team-handbook.md (hunk 7 above)
Comment (jasonblais) [reply operations/research-and-development/product/product-technical-writing-team-handbook.md]: Much better @justinegeffen!

File: operations/research-and-development/product/product-technical-writing-team-handbook.md [hunk 8]
```diff
@@ -40,29 +40,77 @@ The Technical Writing team focuses primarily on the User Guide, Admin Guide, and
...
+* End User Guide - Mobile and desktop app: Eric Sethna (@esethna)
+* End User Guide - Other: Adam Clarkson (@adamjclarkson)
+* Integrations: Aaron Rothschild (@aaronrothschild)
+* Handbook and Process: Jason Blais (@jasonblais)
+
+**Dev Review**
+* Admin Guide - Enterprise: Scott Bishel (@sbishel)
+* Admin Guide - Non-enterprise: Joram Wilander (@jwilander)
+* End User Guide - Mobile and desktop app: Elias Nahum (@enahum)
+* End User Guide - Other:
```
Comment (justinegeffen) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md]:
```suggestion
* End User Guide - Other: Dean Whillier (@deanwhillier)
```

**Review by jasonblais — APPROVED** [review]
Thank you!

**Pull request merged**